KarateTraining/src/test/java/karate-config.js

```diff
@@ -0,0 +1,16 @@
+function fn() {
+  var env = karate.env; // get system property 'karate.env'
+  karate.log('karate.env system property was:', env);
+  if (!env){
+    env = 'qa';
+    }
+
+  var jsonConfig = read('classpath:src/test/resources/environments.json');
+
+  var config = {
+        env: env,
+        baseUrl: jsonConfig[env].baseUrl
```

Comment:
do you need the env value on the config object? I think you must export `jsonConfig[env]`